script.litebox/default.py

```diff
@@ -0,0 +1,179 @@
+import os, sys, re
+import xbmc
+import xbmcgui
+import xbmcaddon
+import xbmcvfs
+from functools import reduce
+from threading import Thread
+ADDON =             xbmcaddon.Addon()
+ADDON_VERSION =     ADDON.getAddonInfo('version')
+ADDON_LANGUAGE =    ADDON.getLocalizedString
+ADDON_PATH =        ADDON.getAddonInfo('path')
+ADDON_ID =          ADDON.getAddonInfo('id')
+ADDON_DATA_PATH =   os.path.join(xbmcvfs.translatePath("special://profile/addon_data/%s" % ADDON_ID))
```

Comment:
not a showstopper, but in the future consider using:
xbmcvfs.translatePath(ADDON.getAddonInfo('profile'))